src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -462,7 +462,8 @@
           command-chan (async/chan
                          (queue/sorted-command-buffer
                           max-enqueued
-                          #(cmd/update-counter! :invalidated %1 %2 inc!)))
+                          #(cmd/update-counter! :invalidated %1 %2 inc!)
+                          #(cmd/update-counter! :ignored %1 %2 inc!)))
```

Comment:
I'd like to register my dislike of positional params.

Reply:
https://stuartsierra.com/2019/09/15/clojure-donts-numbered-parameters

please change this to `(fn [...] ...)` :grin:

Reply:
cool, that's a fair point. I'll update both counters to not use them in a separate commit